cli/src/semgrep/rule.py

```diff
@@ -58,6 +58,8 @@ def resolve_language_string(language_str: str) -> Language:
 
         # add typescript to languages if the rule supports javascript.
         # TODO: Move this hack to lang.json
+        # coupling: if you move this hack, also fix
+        # Core_runner.add_typescript_to_javascript_rules_hack
```

Comment:
in the end what did you decide with @kopecs and @ajbt200128 who I think also added such a fix no?

Reply:
Emma and I discussed it, and at least for me, I'm going to use her fix

Reply:
Going through my github backlog. Sounds like nothing needed from me?